governance/tag-governance.md

```diff
@@ -0,0 +1,541 @@
+
+## Technical Advisory Groups (TAGs)
+
+A CNCF TAG will oversee and coordinate the interests pertaining to a logical area of needs across
+projects, other CNCF groups, and users. They are long-lived groups that report directly to the TOC.
+Areas covered by TAGs may be vertically focused on industry problem domains that align with the
+projects within the CNCF and/or horizontal that support the other TAGs or the projects themselves.
+
+**Examples:**
+- **Vertical:** Infrastructure, Workloads  
+- **Horizontal:** Security & Compliance, Operational Resilience
+
+### Goals
+
+- Strengthen the project ecosystem to meet the needs of end users and project contributors.  
+- Serve as a “bridge” between projects to surface common problems and drive alignment.  
+- Identify gaps in the CNCF project portfolio. Find and attract projects to fill these gaps.  
+- Reduce some project workload on TOC while retaining executive control & tonal integrity with this
+  elected body.  
+- Provide technical expertise during the evaluation of projects to join the CNCF and as they move
+  levels.  
+- Educate and inform users with unbiased, effective, and practically useful information scoped to
+  cloud native.  
+- Focus attention & resources on helping foster project maturity, systematically across CNCF
+  projects.
+
+### Requirements
+
+- MUST have a charter that outlines scope, potential overlaps with other TAGs, and any operational
+  deviations from TAGs.  
+- MUST enumerate any roles within the TAG and the responsibilities of each along with any role
+  membership requirements and lifecycle.  
+- MUST maintain TAG “metadata” - a list of leads, subprojects, and initiatives.  
+- Where applicable, MUST participate in Project Reviews & Contributor Strategy TOC subprojects.  
+- If applicable, MUST document services provided to other TAGs or projects.  
+- MUST hold at least one regularly scheduled *public* monthly meeting, recorded and uploaded to
+  YouTube.  
+- MUST provide periodic updates to the TOC and TOC liaisons on their health, subprojects, and
+  initiatives.  
+- SHOULD provide timely review and completion of project and TOC requests related to the moving
+  levels process.  
+- SHOULD provide regular ecosystem updates on the domain, such as through the regular CNCF community
+  meeting, KubeCon presentations, or to the TOC directly.  
+- SHOULD organize KubeCon CloudNativeCon presence; such as CFPs, Project Pavillion booths etc.  
+- SHOULD form and document subprojects to organize specific ongoing work or services.  
+- SHOULD produce well researched and informed technical white papers, best practices, and other 
+  ecosystem supportive assets.  
+- SHOULD surface gaps or ecosystem trends to related projects.  
+- SHOULD check in with graduated and incubating projects to understand their health, features,
+  changes, etc. and notify the TOC of anything significant, any gaps, novelty, interesting,
+  concerning.
```

Comment:
1. Are all incubating and graduated projects expected to be overseen by all tags? 
1. How frequently is the TAG expected to gain this comprehensive understanding?
1. What is the accountability mechanism or success metric here? (I recognize that this is a "should" entry, but how will the TOC know if a TAG is doing what it _should_ be doing?)